.github/workflows/image-arm.yaml

```diff
@@ -57,6 +57,84 @@ jobs:
           # end of optional handling for multi line json
           echo "::set-output name=matrix::{\"include\": $content }"
 
+  build-nvidia-base:
```

Comment:
the nvidia base image will only be built if its dockerfile changed or if it's a release (see release-arm.yaml). This will save some runners time and I would actually like to do something similar with all other base images so I created a ticket for this https://github.com/kairos-io/kairos/issues/2034